tests/paho_test.py

```diff
@@ -25,13 +24,26 @@ class TestError(Exception):
     def __init__(self, message="Mismatched packets"):
         self.message = message
 
+
+def bind_to_any_free_port(sock) -> int:
```

Comment:
❤️ I will no longer have strange error when I run a real broker on port 1888 and run paho test.